JwtMinimalApi/Controllers/AuthController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using JwtMinimalApi.Models;
+using JwtMinimalApi.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace JwtMinimalApi.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class AuthController : ControllerBase
+{
+    private readonly JwtTokenGenerator _tokenGenerator;
+
+    public AuthController(JwtTokenGenerator tokenGenerator)
+    {
+        _tokenGenerator = tokenGenerator;
+    }
+
+
+    [HttpPost("login")]
+    public IActionResult Login(User requestUser)
```

Comment:
```suggestion
    public IActionResult Login(User userRequest)
```

Note: You can also have two different models for the user, one for the DTO layer and one for the domain/infrastructure layer.